src/taxes/pages/TaxClassesPage/TaxClassesPage.tsx

```diff
@@ -24,23 +33,40 @@ import {
 import { parseQuery } from "@saleor/orders/components/OrderCustomerAddressesEditDialog/utils";
 import { getById } from "@saleor/orders/components/OrderReturnPage/utils";
 import { taxesMessages } from "@saleor/taxes/messages";
-import { getDefaultTaxRateInCountry } from "@saleor/taxes/utils/utils";
 import React from "react";
 import { FormattedMessage, useIntl } from "react-intl";
 
 import TaxInput from "../../components/TaxInput";
+import TaxClassesForm from "./form";
 import { useStyles } from "./styles";
 import TaxClassesMenu from "./TaxClassesMenu";
 
 interface TaxClassesPageProps {
   taxClasses: TaxClassFragment[] | undefined;
   selectedTaxClassId: string;
   handleTabChange: (tab: string) => void;
+  savebarState: ConfirmButtonTransitionState;
+  disabled: boolean;
+  onCreateNewButtonClick: () => void;
+  onTaxClassDelete: (id: string) => SubmitPromise;
+  onTaxClassCreate: (input: TaxClassCreateInput) => SubmitPromise;
+  onTaxClassUpdate: (id: string, input: TaxClassUpdateInput) => SubmitPromise;
```

Comment:
that sound like scoped responsibility, you can move out that logic to the dedicated component, avoid props-drilling